src/Uno.UI/UI/Xaml/Internal/DependencyPropertyHelper.cs

```diff
@@ -0,0 +1,216 @@
+#nullable enable
+
+using Microsoft.UI.Xaml;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Uno.UI.Xaml.Core;
+
+/// <summary>
+/// The goal of this class is to provide a set of helper methods to work with <see cref="DependencyProperty"/> from
+/// external projects.
```

Comment:
I don't see why we need this. Reflection (or even better, UnsafeAccessorAttribute) could still be used to access such implementation details without the need for a new class

Reply:
Wouldn't that mean it would be easier to make unintended breaking changes in Uno that would break the users? This adds a layer of indirection that could help maybe

Reply:
> I don't see why we need this. Reflection (or even better, UnsafeAccessorAttribute) could still be used to access such implementation details without the need for a new class

@Youssef1313 That's funny... I think the need for this is very clear now ;-)